keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/ConvertExampleFilesTest.kt

```diff
@@ -38,62 +42,77 @@ class ConvertExampleFilesTest : JUnit5Minutests {
   fun tests() = rootContext<Unit> {
 
     context("cluster") {
-      mapper.registerSubtypes(NamedType(ClusterSpec::class.java, "cluster"))
+      mapper.registerSubtypes(NamedType(ClusterSpec::class.java, "$SPINNAKER_EC2_API_V1/cluster"))
       val file = this.javaClass.getResource("/examples/cluster-example.yml").readText()
 
       test("yaml can be parsed") {
         expectCatching {
-          mapper.readValue<SubmittedResource<ClusterSpec>>(file)
-        }.succeeded()
+          mapper.readValue<SubmittedResource<*>>(file)
+        }
+          .succeeded()
+          .get { spec }
+          .isA<ClusterSpec>()
```

Comment:
I made this change to all the tests in this class so that it also tests we can identify the type correctly. If we give Jackson the generic type in the `readValue` call it doesn't exercise our type id logic.